<launch>

    <arg name="project" default="lio_sam"/>
    <arg name="param_file" default="$(find lio_sam)/config/params.yaml"/>
    <arg name="enable_rviz" default="true"/>

    <!-- Parameters -->
    <rosparam file="$(arg param_file)" command="load" />

    <!--- LOAM -->
    <include file="$(find lio_sam)/launch/include/module_loam.launch" />

    <!--- Robot State TF -->
    <!-- <include file="$(find lio_sam)/launch/include/module_robot_state_publisher.launch" /> -->

    <!--- Run Navsat -->
    <include file="$(find lio_sam)/launch/include/module_navsat.launch" />

    <!--- Run Rviz-->
    <include file="$(find lio_sam)/launch/include/module_rviz.launch" if="$(arg enable_rviz)" />

</launch>
